src/test/java/com/auth0/client/mgmt/LogEventsEntityTest.java

```diff
@@ -106,7 +106,7 @@ public void shouldListLogEventsWithQuery() throws Exception {
         assertThat(recordedRequest, hasMethodAndPath("GET", "/api/v2/logs"));
         assertThat(recordedRequest, hasHeader("Content-Type", "application/json"));
         assertThat(recordedRequest, hasHeader("Authorization", "Bearer apiToken"));
-        assertThat(recordedRequest, hasQueryParameter("q", "email%3A%5C*%40gmail.com"));
+        assertThat(recordedRequest, hasQueryParameter("q", "email:\\*@gmail.com"));
```

Comment:
I wonder if this change on behavior could be a breaking change??

Reply:
@lbalmaceda
This is my confession:
I'm not a real contributor to auth0...
I'm just suffering from dependency convergence. We're using Zipkin for tracing reporting, and it's already using version 3.14.3 of okhttp3.
For some reason, the auth0 project, which we're also using, is still back on version 3.9.1.
I fear these 2 versions are not compatible.
So I tried to just upgrade the okhttp3 version but found out that some tests are breaking, probably because of changes made between these versions handling URL encoding.
I'll be happy if someone who really understands what is the use of okhttp3 for auth0 will be able to handle this version bump.
I think it's important for auth0 to remain relevant for users, to always try and bump to the latest dependencies versions. Consider using @dependabot for this.
Could you please help with this?
Thanks a lot,
Uziel